runtime/calamari/src/assets_config.rs

```diff
@@ -58,6 +63,12 @@ impl pallet_assets::Config for Runtime {
     type Freezer = ();
     type Extra = ();
     type WeightInfo = weights::pallet_assets::SubstrateWeight<Runtime>;
+    type RemoveItemsLimit = ConstU32<1000>;
+    type AssetIdParameter = CalamariAssetId;
+    type CreateOrigin = AsEnsureOriginWithArg<EnsureSigned<AccountId>>;
```

Comment:
Maybe make root instead. Doesn't really matter as all calls are currently filtered but a little bit of a footgun if we remove the filter.

Reply:
Yeah this doesn't make sense.

Reply:
EnsureNever makes more sense since this CreateOrigin should never be used, and Root uses ForceCreate